### PR TITLE
:bento: Dynamic missions

### PR DESCRIPTION
**Pull request by Anny85-code**
### In this milestone I;
- [x] Render missions dynamically in the my-project page.

### CODE REVIEW

File: src/pages/MyProfile/MyProfile.js [hunk 1]
```diff
@@ -3,39 +3,46 @@ import TableItem from '../../Components/Table-Item/TableItem';
...
+  //   },
+  //   {
+  //     name: 'Mexico',
+  //     id: '4',
+  //   },
+  //   {
+  //     name: 'Cuba',
+  //     id: '5',
+  //   },
+  // ];
```
Comment (matovu-farid) [inline src/pages/MyProfile/MyProfile.js]: Please remove these comments

**Review by matovu-farid — CHANGES_REQUESTED** [review]
You have done a great job so far
However, there are a few changes to address
Please look at the comments below

File: src/pages/MyProfile/MyProfile.js (hunk 1 above)
Comment (Anny85-code) [reply src/pages/MyProfile/MyProfile.js]: Thanks for your review, changes will be implemented

**Review by matovu-farid — APPROVED** [review]
Your task is complete. There is nothing else to say other than it is time to merge it
✔️  No linter errors
✔️  All project requirements met

**Pull request merged**